src/autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h

```diff
@@ -6,7 +6,7 @@
 
 #pragma once
 
-#include "VerletListsCellsHelpers.h"
+#include "autopas/containers//verletListsCellBased/verletListsCells/VerletListsCellsHelpers.h"
```

Comment:
```suggestion
#include "autopas/containers/verletListsCellBased/verletListsCells/VerletListsCellsHelpers.h"
```
one `/` too many.
It's actually in the same directory, so not pretending the path is ok.